<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlFluidic</class>
 <widget class="QMainWindow" name="ControlFluidic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fluidic circuit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>210</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Auto</string>
     </attribute>
     <widget class="QSlider" name="horizontalSlider_flush">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2000</number>
      </property>
      <property name="maximum">
       <number>4000</number>
      </property>
      <property name="singleStep">
       <number>200</number>
      </property>
      <property name="pageStep">
       <number>200</number>
      </property>
      <property name="sliderPosition">
       <number>3500</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>200</number>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_sample">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>300</number>
      </property>
      <property name="maximum">
       <number>1500</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="pageStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>900</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>100</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sample">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sample</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_flush">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Flush</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>120</width>
        <height>5</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/fluidic/colorbar_flush.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>120</width>
        <height>5</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons.qrc">:/fluidic/colorbar_sample.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>2 ml/min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>4 ml/min</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>105</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0.3 ml/min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>105</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>1.5 ml/min</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>200 1/s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>1000 1/s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_hellmanex">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Flush w/ Hellmanex III</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>valve</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_direction">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>155</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>direction</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_brake">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>155</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>brake</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>on/off</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_apply">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>155</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_status_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>150</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Revolutions per minute (RPM)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_status_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>30</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>RPM</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_rpm">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_pulsationReduction">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>125</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>pulsation reduction</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Plot" name="qwtPlot_stats">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>260</y>
      <width>210</width>
      <height>140</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QLabel" name="label_rpm">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>8</y>
       <width>60</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00 RPM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>235</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;is connected?&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>pushButton_connect</zorder>
   <zorder>qwtPlot_stats</zorder>
   <zorder>label_status</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plot</class>
   <extends>QFrame</extends>
   <header>plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
